$NetBSD: patch-aa,v 1.1 2000/02/14 16:23:32 agc Exp $

Check which OS we're on, and use the correct LIBS and
CPPFLAGS appropriately.

--- Makefile	2000/02/14 16:14:21	1.1
+++ Makefile	2000/02/14 16:15:16
@@ -4,7 +4,13 @@
 CFLAGS=
 
 # For Solaris 2.5.1 and above, uncomment the following line.
+
+OPSYS!=	uname -s
+
+.if ${OPSYS} == "SunOS"
+CPPFLAGS=
 LIBS=-lresolv -lsocket -lnsl
+.endif
 
 default:
 	@clear
